examples/simple-file-server/README.md

```diff
@@ -0,0 +1,74 @@
+# Simple File Server
+
+## Overview
+
+This guide explains how to put and get a file to/from a UDP server using Ballerina.
+The following figure illustrates a high-level design diagram of the complete use case.
+
+![Sending a File with UDP](./simple-udp-server.jpg)
+
+The below are the detailed explanations of each of the steps of this example.
```

Comment:
Maybe we can have another topic saying `Execution Order`

Reply:
Added the topic.